concent_api/utils/helpers.py

```diff
@@ -185,22 +200,29 @@ def calculate_subtask_verification_time(report_computed_task: message.ReportComp
     """
     assert isinstance(report_computed_task, message.ReportComputedTask)
 
-    mdt = maximum_download_time(
-        size=report_computed_task.size,
-    )
-    ttc_dt = datetime.datetime.utcfromtimestamp(
-        report_computed_task.task_to_compute.timestamp,
-    )
-    subtask_dt = datetime.datetime.utcfromtimestamp(
-        report_computed_task.task_to_compute.compute_task_def['deadline'],
-    )
-    subtask_timeout = subtask_dt - ttc_dt
-
-    return int(
-        (4 * settings.CONCENT_MESSAGING_TIME) +
-        (3 * mdt.total_seconds()) +
-        (0.5 * subtask_timeout.total_seconds())
-    )
+    if settings.CUSTOM_PROTOCOL_TIMES:
+        mdt = maximum_download_time(
+            size=report_computed_task.size,
+        )
+        ttc_dt = datetime.datetime.utcfromtimestamp(
+            report_computed_task.task_to_compute.timestamp,
+        )
+        subtask_dt = datetime.datetime.utcfromtimestamp(
+            report_computed_task.task_to_compute.compute_task_def['deadline'],
+        )
+        subtask_timeout = subtask_dt - ttc_dt
+
+        return int(
+            (4 * settings.CONCENT_MESSAGING_TIME) +
+            (3 * mdt.total_seconds()) +
+            (0.5 * subtask_timeout.total_seconds())
+        )
+    else:
+        return int(
+            subtask_verification_time(
+                report_computed_task
+            ).total_seconds()
```

Comment:
I think this is short enough to be on a single line. It's acceptable like this but it does not have multiple parameters, long mathematical expressions or really anything that would require it to be split like that.

Reply:
:+1: